_drafts/CocoaPods-1.9.0-beta.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title:  "CocoaPods 1.9 Beta has arrived!"
+author: amorde
+categories: cocoapods releases
+---
+
+_CocoaPods 1.9_ adds support for .xcframeworks, configuration based dependencies for pod authors, and other ehancements and bug fixes!
+
+<!-- more -->
+
+This release includes several quality-of-life enchancements in addition to important bug fixes.
+
+## XCFramework Support
+
+With the release of Xcode 11, Apple introduced a new bundle format using the `.xcframework` file extension. This format allows multiple copies of a framework compiled for different architectures and platforms to be combined into a single structure.
```

Comment:
xcframework here is wrapped around backticks but not anywhere else, which one should we use?

Reply:
I think it should be for any time it's used as the file extension, and written as "XCFramework" otherwise. I'll check for that